docs/OpenID4VPCrossDeviceFlow.md

```
@@ -1,56 +1,126 @@
+
+- Result Format Configuration
+
+  A new attribute summariseResults has been introduced.
+
+  This attribute determines the format of the response returned by the SDK.
+
+  - When summariseResults = true
+
+    The SDK returns a simplified, high-level response:
+
+    ```json
```

markdownlint-cli2 (0.22.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)